ciao-deploy/deploy/nodes.go

```diff
@@ -188,31 +188,36 @@ func SetupNodes(ctx context.Context, sshUser string, networkNode bool, hosts []s
 }
 
 func teardownNode(ctx context.Context, hostname string, sshUser string) error {
+	var errOut error
 	tool := "ciao-launcher"
 	fmt.Printf("%s: Stopping %s\n", hostname, tool)
 	err := SSHRunCommand(ctx, sshUser, hostname, fmt.Sprintf("sudo systemctl stop %s", tool))
 	if err != nil {
-		return errors.Wrap(err, "Error stopping tool on node")
+		errOut = errors.Wrap(err, "Error stopping tool on node")
+		fmt.Fprintln(os.Stderr, err.Error())
```

Comment:
Shouldn't this be

fmt.Fprintln(os.Stderr, errOut.Error())

Reply:
Yes, thanks. Fixed!